lib/client.js

```diff
@@ -534,10 +536,11 @@ class Parser extends HTTPParser {
     const { client, socket, timeout } = this
 
     const request = client[kQueue][client[kRunningIdx]]
+    const headersTimeout = getRunningRequestOpts(client, kHeadersTimeout)
```

Comment:
```suggestion
    const headersTimeout = request.headersTimeout != null ? request.headersTimeout  : client[kHeadersTimeout]
```

Reply:
I thought symbols are used for optimization since better to have a reference always to the same "symbol" rather than creating a new "string" object every time, am I wrong?

Reply:
Symbols are just to hide things from the user, i.e. instead of `private` which exists in many other languages. It's not for optimization.